### PR TITLE
Fixed documentation

### PR DESCRIPTION
**Pull request by tonydavis629**
# Pull Request Template

## Description

Fixed some minor typos as well as fixing formatting for references, which are not displaying correctly in deepchem.readthedocs.io.

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [x] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [ ] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.22.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [ ] Run `flake8 <modified file> --count` and check no errors
  - [ ] Run `python -m doctest <modified file>` and check no errors
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New unit tests pass locally with my changes
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/models/keras_model.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def __init__(self,
                log_frequency: int = 100,
                wandb_logger: Optional[WandbLogger] = None,
                **kwargs) -> None:
-    """Create a new KerasModel.
+    """Create a new Keras Model.
```
Comment (rbharath) [inline deepchem/models/keras_model.py:140]: Should be `KerasModel` (since it's the name of the class)

File: deepchem/models/keras_model.py [hunk 2]
```diff
@@ -1183,7 +1183,7 @@ def load_from_pretrained(self,
     `value_map` is created. `assignment_map` is a dictionary mapping variables
     from the `source_model` to the current model. If no `assignment_map` is
     provided, one is made from scratch and assumes the model is composed of
-    several different layers, with the final one being a dense layer. include_top
+    several different layers, with the final one being a dense layer. 'include_top'
```
Comment (rbharath) [inline deepchem/models/keras_model.py:1186]: Should be `include_top` (that is, use ` instead of ' for quoting)

**Review by rbharath — COMMENTED** [review]
Couple of minor typos but otherwise looking good! Can merge once you fix

**Comment by tonydavis629** [comment]
I see, I've made the corrections. I quoted `include_top` since the other parameters are quoted.

**Review by rbharath — APPROVED** [review]
LGTM! (Looks Good to Me)

Will merge in once tests pass

**Comment by rbharath** [comment]
Congrats on your first merged PR @tonydavis629!

**Pull request merged**